README.md

```diff
@@ -27,36 +26,65 @@ With "read the manual" fatigue and lengthy selection processes wearing on us eve
+- [sgv2-graphqlapi](apis/sgv2-graphqlapi): API implementation for exposing Cassandra data over GraphQL
+- [sgv2-docsapi](apis/sgv2-docsapi): API implementation for exposing Cassandra data over a Document API
 
-    - cql: API implementation for the Cassandra Query Language
-    - restapi: API implementation for exposing Cassandra data over REST
-    - graphqlapi: API implementation for exposing Cassandra data over GraphQL
+Each API Service contains its own integration test suite that tests it against the coordinator node and supported Cassandra backends. There is also a [sgv2-quarkus-common](apis/sgv2-quarkus-common) module containing utilities that may be used by all Java/Quarkus based API services.
 
-- **Persistence Services**: Responsible for implementing the coordination layer to execute requests passed by API services to underlying data storage instances.
+### Coordinator Node
+Coordinator nodes participate as non-data storing nodes in the backing Cassandra cluster, which enables them to read and write data more efficiently. Stargate Coordinator nodes can also be scaled independently. Coordinator nodes expose gRPC and CQL interfaces for fast access by client applications. The following are the key modules comprising the coordinator and its exposed interfaces:
 
-    - persistence-api: Interface for working with persistence services
-    - persistence-common: Utilities shared by the persistence services
-    - persistence-cassandra-3.11: Joins C* 3.11 cluster as coordinator-only node (does not store data),
-    mocks C* system tables for native driver integration,
-    executes requests with C* storage nodes using C* QueryHandler/QueryProcessor,
-    converts internal C* objects and ResultSets to Stargate Datastore objects.
-    - persistence-cassandra-4.0: (same as above but for Cassandra 4.0)
+- [core](core): Common classes used throughout the other coordinator modules
+- [cql](cql): API implementation for the Cassandra Query Language
+- [grpc](grpc): fast CQL over gRPC implementation (HTTP-based interface equivalent to CQL performance)
+- [bridge](bridge): gRPC-based interface used by API services
+- [health-checker](core): HTTP endpoints useful for health checking coordinator nodes
+- [metrics-jersey](core): metrics collection for the coordinator node and its exposed interfaces
+- [stargate-starter](stargate-starter): the main Java application used to start the coordinator via the `starctl` script
 
-- **Authentication Services**: Responsible for authentication to Stargate
+#### Persistence Services
+Stargate coordinator nodes support a pluggable approach for implementing the coordination layer to execute requests passed by API services and other interfaces to underlying data storage instances. Persistence service implementations are responsible handling and converting requests to database queries, dispatching to persistence, and returning and serving responses.
```

Comment:
Could also mention that they expose unified interface to hide differences between various Cassandra versions, so that API services operate against "single" backend, essentially (although TBH with v2 I guess this is more due to Bridge exposing unified API).

Reply:
added detail